ansible/roles_ocp_workloads/ocp4_workload_performance_monitoring/files/grafana_role.yml

```diff
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: route-viewer
+rules:
+  - verbs:
+      - get
+      - list
+      - watch
+    apiGroups:
+      - ''
+      - route.openshift.io
+    resources:
+      - routes
```

Comment:
our checks might freak out if no newline at EOF

Reply:
Do you mean lint? Because that was already checked.